components/firebase_admin_sdk/actions/create-realtime-db-record/create-realtime-db-record.mjs

```diff
@@ -0,0 +1,38 @@
+import common from "../common/base.mjs";
+
+export default {
+  ...common,
+  key: "firebase_admin_sdk-create-realtime-db-record",
+  name: "Create Firebase Realtime Database Record",
+  description: "Creates or replaces a child object within your Firebase Realtime Database. [See the docs here](https://firebase.google.com/docs/reference/js/database#update)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    ...common.props,
+    path: {
+      propDefinition: [
+        common.props.firebase,
+        "path",
+      ],
+    },
+    data: {
+      propDefinition: [
+        common.props.firebase,
+        "data",
+      ],
+      description: "Multiple property-value pairs that will be written to the Database together",
+    },
+  },
+  methods: {
+    async getResponse() {
+      return this.firebase.createRealtimeDBRecord(this.path, this.data);
+    },
+    emitSummary($) {
+      $.export("$summary", "Successfully created record");
+    },
+    async deleteApp() {
+      this.firebase.deleteApp();
+      await new Promise((resolve) => setTimeout(resolve, 5000));
```

Comment:
This is the only action that accesses the Firebase Realtime Database instead of the Firestore Database and uses `admin.app().database()`. The others finish without errors, but without the `sleep` here, this will display a message saying the code was still running when it stopped.